sdk/core/azure-core-tracing-opentelemetry/CHANGELOG.md

```diff
@@ -6,10 +6,14 @@
 
 ### Breaking Changes
 
+- The `Azure::Core::Tracing::OpenTelemetry::OpenTelemetryProvider` type can only be instantiated via a factory method: `OpenTelemetryProvider::Create`.
```

Comment:
If we're releasing this as GA, let's repeat the Beta 1 message for this release changelog instead of listing breaking changes.